### DISCUSSION TITLE
Does not work on Debian and MacOS M1

### DISCUSSION THREAD

**Discussion by Nemo-Illusionist** [dlemstra/Magick.NET#1475] [Help]
Hello.
I use a library for image compression.
Everything works on Windows, but on Debian server and MacOS M1 laptop the images are turned into 1 pixel.

Use `<PackageReference Include="Magick.NET-Q8-AnyCPU" Version="13.3.0" />`

```ch
using var image = new MagickImage(file.Stream)
var defines = new JpegWriteDefines { Extent = 700 };
image.Settings.SetDefines(defines);
image.FilterType = FilterType.Lanczos2Sharp;
image.ColorSpace = ColorSpace.sRGB;
image.Resize(1920, 0);
image.UnsharpMask(0, 0.45, 0.45, 0);

image.Write(memoryStream);
memoryStream.Position = 0;
```

**Comment by dlemstra** [comment]
Can you create a Dockerfile that can reproduce this on `Debian`? There are Linux unit tests and that all seem to pass here. And I don't have a M1 so I cannot help you with that. But maybe if we figure out the `Debian` part we will also resolve your MacOS issue?
